src/DynamoCoreWpf/DynamoCoreWpf.csproj

```diff
@@ -1491,6 +1491,10 @@
     <EmbeddedResource Include="Views\GuidedTour\HtmlPages\Resources\DynamoPackages_1_1.png" />
     <Resource Include="UI\Images\default_info_16px.png" />
     <Resource Include="UI\Images\clock-icon.png" />
+    <Resource Include="UI\Images\question-default-16px.png" />
+    <Resource Include="UI\Images\question-hover-16px.png" />
+    <Resource Include="UI\Images\info-default-16px.png" />
+    <Resource Include="UI\Images\info-hover-16px.png" />
```

Comment:
I am surprised that we do not have these images already in the solution, are these new ones?

Reply:
Yes, They are new, We had already some similar but they weren't the same according to the expected design https://www.figma.com/file/dyhgWVlxDPJvOUTL07hUxa/Dynamo-Splash-Screen?node-id=207%3A91295